lib/src/empty_unmodifiable_set.dart

```diff
@@ -4,65 +4,44 @@
 
 import 'dart:collection';
 
+import 'package:collection/collection.dart';
+
 import 'unmodifiable_wrappers.dart';
 
-// Unfortunately, we can't use UnmodifiableSetMixin here, since const classes
-// can't use mixins.
-/// An unmodifiable, empty set that can have a const constructor.
+/// An unmodifiable, empty set which can be constant.
 class EmptyUnmodifiableSet<E> extends IterableBase<E>
```

Comment:
[out of scope for this PR] Is this class buying value over `const <E>{}`? Should we consider deprecating it?

Reply:
Probably not worth it any more. It's a valid superclass, but I can't see for what.
Yes, we should consider deprecating it.